documentation/example-session.md

```diff
@@ -7,38 +7,43 @@ const arc = new Arc({
   web3Provider: 'https://url.to.web3.node/',
 })
 
-// get an Observable list of DAOs
+// get an Observable for list of DAOs
 const daoObservable = arc.daos()
-// or just get the list itself
-const daos = await arc.daos().toPromise()
+// we get the first return item from the obervable
+const { first } = require('rxjs/operators')
+const daos = await arc.daos().pipe(first()).toPromise()
 
 // you can query for a particular DAO
 arc.daos({ name: 'mydao'})
 
-const dao = await arc.dao(daos[0].address)
+// or if you know the address, just create a new DAO object like this:
+const dao = arc.dao(daos[0].address)
```

Comment:
Why did you remove the `await`?

Reply:
because it is not asynchronous, so it is not needed